app/models/user.rb

```diff
@@ -1,13 +1,15 @@
 class User < ApplicationRecord
+  attribute :terms_and_service, default: false
 
   normalizes :email_address, with: ->(e) { e.strip.downcase }
 
-  attribute :terms_and_service, default: false
-
   has_secure_password
+  validates :password, presence: true, password: true
+  validates :password_confirmation, presence: true
 
   has_many :sessions, dependent: :destroy
 
-  validates :terms_and_service, acceptance: { message: "^You cannot register without accepting the terms of use." }
-
+  validates :email_address, uniqueness: true
+  validates :email_address, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
+  validates :terms_and_service, acceptance: true
```

Comment:
https://github.com/m8d6/chat-app/pull/11/files#r1984700712 bu yorumum geçerli bu dosya için hala :)